feature/swipe to complete checkin in guestlist

mhlangagc opened the pull request:
### Closes Issues:
- #81 
- #82 

### Description:



### Release video link
https://cl.ly/580f69bf45ed


## Release Details:
### Pods Installed
- pod 'SwipeCellKit', '2.5.4'

Krakaw commented on Big Neon Core/Services/DatabaseService/DatabaseService.Scan.swift:
```diff
@@ -152,8 +152,8 @@ extension DatabaseService {
         
-        
         let apiURL = APIService.fetchEvents(eventID: eventID, changesSince: changeSince, page: nil, limit: nil, query: nil)
+        print(apiURL)
```
Do we need this print here?

Krakaw commented on Big Neon Core/Utilities/DateConfig.swift:
```diff
@@ -55,9 +55,15 @@ final public class DateConfig {
     public class func serverDateFromDate(stringDate: Date) -> String? {
         let dateFormatter = DateFormatter()
-        dateFormatter.dateFormat = "yyyy-MM-dd'T'HH:mm:ss"
+        dateFormatter.dateFormat = "yyyy-MM-dd'UTC'HH:mm:ss"
```
Does the formatter require the timezone in place of the T as that is a string literal?

Krakaw commented on Big Neon/Screens/Explore/GuestListViewController/GuestsListViewModel.swift:
```diff
@@ -77,21 +68,23 @@ final class GuestsListViewModel {
         }
         
-        BusinessService.shared.database.fetchUpdatedGuests(forEventID: eventID, changeSince: date) { [weak self] (error, guestsFetched) in
+        print(date)
```
Print required?

Krakaw commented on Big Neon/Screens/Explore/GuestListViewController/GuestsListViewModel.swift:
```diff
@@ -77,21 +68,23 @@ final class GuestsListViewModel {
                     return
                 }
                 
+                print(guests)
```
Print?

Krakaw commented on Big Neon/Screens/Explore/TicketScannerViewController/ScannerViewController.AutoScan.swift:
```diff
@@ -6,12 +6,18 @@ import AVFoundation
             DispatchQueue.main.async {
                 if fromGuestTableView == true {
-                    self?.guestListVC?.reloadGuests(atIndex: atIndexPath!)
+                    if self?.guestListVC?.isSearching == true {
```
Should we set local variables for the `self?.guestListVC?` to keep it DRY?

Krakaw commented:
Minor comments, otherwise looks good.

mhlangagc commented:
@Krakaw I've updated the swipe to checkin feature and removed the bugs you picked up. If you possibly can, please review the app on a device (I can show it to you on mine). The swipe to checkin feature doesn't swipe the whole way (though it could) but there is haptic feedback to let the user know they have swiped in the user.
